src/reporter/format-xspec-report.xsl

```diff
@@ -516,6 +523,28 @@
       <xsl:value-of select="$components" separator="{if ($insert-labels) then ' / ' else '/'}" />
    </xsl:template>
 
+   <xsl:template name="x:report-elapsed" as="element(xhtml:span)?">
+      <!-- Context item is x:report or x:scenario -->
+      <xsl:context-item as="element()" use="required" />
+
+      <xsl:if test="x:timestamp">
+         <xsl:variable name="elapsed" as="xs:dayTimeDuration" select="
+               xs:dateTimeStamp(x:timestamp[@event eq 'end']/@at)
+               - xs:dateTimeStamp(x:timestamp[@event eq 'start']/@at)" />
+         <span>
+            <xsl:if test="self::x:scenario">(</xsl:if>
+            <xsl:text>Elapsed: </xsl:text>
+            <span class="elapsed">
+               <xsl:value-of select="$elapsed div xs:dayTimeDuration('PT1S')" />
+            </span>
+            <xsl:text>s</xsl:text>
+            <xsl:if test="self::x:scenario">
+               <xsl:text>) </xsl:text>
+            </xsl:if>
```

Comment:
A few comments about the HTML report:

1. You might consider some ways to make it easier to scan a set of elapsed times visually, when they are inline with the scenario label: for example, write "(0.002 s elapsed)" instead of "(Elapsed: 0.002 s)"; perhaps remove "Elapsed: " entirely; or make "(Elapsed: 0.002 s)" right-justified in its column. Any of these ideas could be a later refinement in a future release. I like the "Elapsed: ..." label at the top of the HTML report, above "Contents", where it is not inline with any label.

2. SI says to put a space between the number and the unit (https://en.wikipedia.org/wiki/Space_(punctuation)#Unit_symbols_and_numbers).

3. The timing notation appears non-bold in the "Contents" section but bold in the detailed section. Is the difference intentional? I like it non-bold so it doesn't compete as much with the scenario label.

Reply:
> 2. SI says to put a space between the number and the unit (https://en.wikipedia.org/wiki/Space_(punctuation)#Unit_symbols_and_numbers).

Fixed in db6cde4d2ac74c6825af6b60b72df20757dd8024.

Reply:
> 1. You might consider some ways to make it easier to scan a set of elapsed times visually, when they are inline with the scenario label: for example, write "(0.002 s elapsed)" instead of "(Elapsed: 0.002 s)"; perhaps remove "Elapsed: " entirely; or make "(Elapsed: 0.002 s)" right-justified in its column. Any of these ideas could be a later refinement in a future release. I like the "Elapsed: ..." label at the top of the HTML report, above "Contents", where it is not inline with any label.

- ccac4b6ab2001d0308c870208b637a64d538a99c right-aligns them by `float: right;`. Float is ugly in general but it will be ok for now. HTML report needs major overhaul anyway. (#92, #919)
- 35c051a8aef30653a9328e4df2fdee8b399185f2 changes the unit (milliseconds).

> 3. The timing notation appears non-bold in the "Contents" section but bold in the detailed section. Is the difference intentional? I like it non-bold so it doesn't compete as much with the scenario label.

Fixed in ccac4b6ab2001d0308c870208b637a64d538a99c.